frontend/express/public/javascripts/countly/vue/components/dropdown.js

```diff
@@ -187,6 +187,123 @@
         }
     }));
 
+    Vue.component("cly-vue-menubox", countlyBaseComponent.extend({
```

Comment:
I think this needs a better name:

-Shouldn't contain "vue" keyword
-Should emphasize the function 

For instance, cly-fields-select